Automattic/wp-calypso
# Change Google My Business Stats Nudge Image

## Specs

Replace the image on the Google My Business stats nudge. Before we were using a Google screenshot, now we are using one of our own stats page. Also a couple CSS tweaks to prevent stretching the text of the image.

### Before
<img width="1049" alt="screen shot 2018-06-20 at 8 00 50 am" src="https://user-images.githubusercontent.com/2810519/41666768-1e63bcbc-7460-11e8-8243-18b8df07b6d8.png">

### After
<img width="1070" alt="screen shot 2018-06-19 at 1 00 33 pm" src="https://user-images.githubusercontent.com/2810519/41621324-2944526c-73c1-11e8-89cc-475730b68ddd.png">

## Testing
1. Boot this branch locally use live branch
2. Navigate to Stats Page of GMB Eligible, but unconnected site
3. Verify that the new  image appears
4. Ponder your feelings on the new screenshot
